src/H5detect.c

```diff
@@ -722,21 +694,9 @@ find_bias(unsigned int epos, unsigned int esize, int *perm, void *_a)
 static void
 print_header(void)
 {
-
-    time_t      now = HDtime(NULL);
-    struct tm  *tm  = HDlocaltime(&now);
-    char        real_name[30];
-    char        host_name[256];
-    int         i;
-    const char *s;
-#ifdef H5_HAVE_GETPWUID
-    struct passwd *pwd = NULL;
-#else
-    int pwd      = 1;
-#endif
-    static const char *month_name[] = {"Jan", "Feb", "Mar", "Apr", "May", "Jun",
-                                       "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
-    static const char *purpose      = "\
+    int                i;
```

Comment:
I'd like to leave the date+time, hostname, etc. information about where the file was generated.  It can be useful provenance information for archiving and debugging.

Reply:
I disagree. It's a lot of platform-specific hax for a "feature" of very limited value. I would be very, very, very surprised if *anyone* in the past twenty years has tried to figure out when they configured HDF5 by peeking in generated H5T files. I can also imagine that some people are not going to be super excited to expose internal machines and usernames via HDF5 source code. And we don't do this in other generated files.